docs/dashboards/custom-chart.md

```diff
@@ -0,0 +1,162 @@
+# Custom Chart
+
+The custom chart is an advanced block that allows you to create custom data visualization by utilizing Vega-Lite.
```

Comment:
- ( this is in a lot of places ) Block names are proper: Custom Chart Block

- Link to Vega-Lite

- Add an image and example block on the dashboard

- You could add another sentence here: "If you would like a specific visualization that's not already a Losant Dashboard Block, the Custom Chart Block gives you the ultimate support for a wide array of visualization types" This first paragraph needs to tell me why I need the block. Check out the Input Controls intro for example: 

```
The Input Controls Block allows you to send commands to devices, or to trigger workflows, directly from a dashboard. The block includes various input components to allow for configuring the payloads sent with those commands or triggers, and those components can also be configured to show the current state of your devices.
```